src/style.scss

```diff
@@ -27,9 +27,24 @@
 //   ...
 // }
 
-.testClass {
-  color:  var(--auro-color-border-error-on-light);
-  border: 1px solid var(--auro-color-border-error-on-light);
-  display: inline-block;
-  padding: var(--auro-text-body-size-default);
+.badge {
+  align-items: center;
+  background-color: var(--auro-color-ui-default-on-light);
```

Comment:
This maybe out of the scope for this MVP, but I feel that we should have a color API much like what we did with alerts and [icon](https://github.com/AlaskaAirlines/auro-icon/blob/master/src/auro-icon.js#L153) to allow for badge color flexibility.

Reply:
Agreed for a future use case, but I also see the potential for certain colors to be limited (like alert). Lets wait until we hear of the next use case. For now, we only need blue.